src/Service.Tests/SqlTests/GraphQLMutationTests/MultipleCreateMutationTests/MsSqlMultipleCreateMutationTests.cs

```diff
@@ -0,0 +1,629 @@
+                WHERE [table0].[id] >= 5001
+                ORDER BY [id] asc
+                FOR JSON PATH,
+                    INCLUDE_NULL_VALUES";
+
+            await ManyTypeMultipleCreateFailsDueToCreatePolicyFailure(expectedErrorMessage, bookDbQuery, publisherDbQuery);
+        }
+
+        [TestMethod]
+        public async Task PointMultipleCreateMutationWithReadPolicyViolationAtRelatedEntity()
+        {
+
+            string expectedResponse = @"{
+                                          ""id"": 5001,
+                                          ""title"": ""Book #1"",
+                                          ""publisher_id"": 2345,
+                                          ""reviews"": {
+                                            ""items"": [
+                                              {
+                                                ""book_id"": 5001,
+                                                ""id"": 5001,
+                                                ""content"": ""Review #1"",
+                                                ""websiteuser_id"": 4
+                                              }
+                                            ]
+                                          }
+                                        }";
+
+            await PointMultipleCreateMutationWithReadPolicyViolationAtRelatedEntity(expectedResponse);
+        }
+
+        [TestMethod]
+        public async Task MultipleCreateMutationWithOneToOneRelationshipDefinedInConfigFile()
+        {
+            string expectedResponse1 = @"{
+                                          ""userid"": 3,
+                                          ""username"": ""DAB"",
+                                          ""email"": ""dab@microsoft.com"",
+                                          ""UserProfile_NonAutogenRelationshipColumn"": {
+                                            ""profileid"": 3,
+                                            ""userid"": 10,
+                                            ""username"": ""DAB"",
+                                            ""profilepictureurl"": ""dab/profilepicture""
+                                          }
+                                        }";
+
+            string expectedResponse2 = @"{
+                                          ""userid"": 4,
+                                          ""username"": ""DAB2"",
+                                          ""email"": ""dab@microsoft.com"",
+                                          ""UserProfile_NonAutogenRelationshipColumn"": {
+                                            ""profileid"": 4,
+                                            ""userid"": 10,
+                                            ""username"": ""DAB2"",
+                                            ""profilepictureurl"": ""dab/profilepicture""
+                                          }
+                                        }";
```

Comment:
relationship between the fields user.username and userprofile.username right? where value is DAB2?

just clarifying becuase userid in each is different.

Reply:
That's correct. The relationship fields are  `User.username` and `User_Profile.username`.  

So, `username` is the same in both the entities whereas `userid` can be different.